modal/io_streams.py

```diff
@@ -79,15 +79,17 @@ def __init__(
         object_id: str,
         object_type: Literal["sandbox", "container_process"],
         client: _Client,
+        raw: bool = True,  # if False, streamed logs are further processed into complete lines.
```

Comment:
"raw" to me implies something about the encoding — would `by_line: bool` make sense?